proto/transaction.proto

```diff
@@ -37,4 +41,6 @@ message Transaction {
     TransactionStatus status = 5;
     string abort_reason = 6;
     repeated string delete_set = 7;
+
+    TransactionRemaster remaster = 8;
 }
```

Comment:
@ctring What are your thoughts on this format for remaster transactions?

Remaster transactions would be detected with `has_field("remaster")`. Keys in the write set are moved to the new master. If the transaction still fills out the normal write set and metadata (for the keys that are moving) then the existing could should be able to handle it.